game-app/game-core/src/test/java/org/triplea/util/LocalizeHtmlTest.java

```diff
@@ -8,8 +8,12 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
+import org.triplea.io.FileUtils;
 
 class LocalizeHtmlTest {
+
+  private final Path path = Path.of("/does/not/exist");
```

Comment:
nit, i'd recommend to just duplicate this. The 'rules' of how to write clean tests are different from other code. Having a test be more self contained is valuable, particularly so when it means less redirection.